DigitalLearningSolutions.Web/Views/RegisterDelegateByCentre/WelcomeEmail.cshtml

```diff
@@ -1,14 +1,13 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.Register.RegisterDelegateByCentre
+﻿@using DigitalLearningSolutions.Data.Services
+@using DigitalLearningSolutions.Web.ViewModels.Register.RegisterDelegateByCentre
 @model WelcomeEmailViewModel
+@inject IClockService ClockService

```

Comment:
Rider's telling me this should be `clockService`

Reply:
Interesting, I have the opposite warning (i.e. that it should be `ClockService`). My Settings > Editor > Code Style > C# > Naming > Properties setting is "UpperCamelCase", which in my version of Rider (2022.1.2, which is the latest available) is the default.

Reply:
Ah good to know, now worries then